airflow-core/src/airflow/observability/stats.py

```diff
@@ -22,11 +22,11 @@
 from collections.abc import Callable
```

Comment:
task sdk might need to have a similar factory method/module

Reply:
I added a copy from the factory methods from `airflow-core` and adjusted the imports.